boards/DIM/Src/App/App_DimWorld.c

```diff
@@ -6,6 +6,7 @@
 
 struct DimWorld
 {
+    uint32_t                  current_ms;
```

Comment:
Perhaps `current_time_ms` just to be a bit more clear?

Reply:
Fixed!